apps/web/google_docs.py

```diff
@@ -0,0 +1,35 @@
+from talon import Context, Module, actions
+
+ctx = Context()
+mod = Module()
+
+apps = mod.apps
+apps.gdocs = """
+tag: browser
+browser.host: docs.google.com
+"""
+
+
+@ctx.action_class("user")
+class FindAndReplace:
+    """Override generic find-and-replace"""
+
+    def find(text: str):
```

Comment:
I don't think you need to implement the find, find_next, find_previous, or paste_match_style actions. Those are already implemented for mac with the same shortcuts in `core/edit/edit_mac.py`.